app/views/donations/_confirm.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= button_to(donation_confirmation_url(donation), method: :post) do %>
```

Comment:
I think that `button_to`'s default method is `:post` and the `method:` option is not needed.